geode-core/src/main/java/org/apache/geode/distributed/internal/InternalLocator.java

```diff
@@ -773,15 +773,19 @@ private void startCache(DistributedSystem system) throws IOException {
     startClusterManagementService();
   }
 
-  private void startClusterManagementService() throws IOException {
+  @VisibleForTesting
+  void startClusterManagementService() throws IOException {
     startConfigurationPersistenceService();
 
-    if (internalCache == null) {
+    InternalCache myCache = this.internalCache;
+    InternalLocator currentLocator = InternalLocator.locator;
+
+    if (myCache == null || currentLocator == null) {
       return;
     }
 
-    clusterManagementService = new LocatorClusterManagementService(locator.internalCache,
-        locator.configurationPersistenceService);
+    clusterManagementService = new LocatorClusterManagementService(currentLocator.internalCache,
```

Comment:
should "currentLocator.internalCache" be changed to "myCache"?

Reply:
yes.  It looks like Aditya Anchuri added that code early in 2019.  I'll change it.